README.md

```diff
@@ -7,12 +7,15 @@ SPDX-License-Identifier: CC-BY-4.0
 <img src="specification/figures/glTF.svg" width="340" height="170" />
 </p>
 
-[![Join the glTF Slack group](https://img.shields.io/badge/chat-on%20slack-blue.svg)](https://khr.io/gltfslack)
 [![Join the forums](https://img.shields.io/badge/discuss-in%20forums-blue.svg)](https://community.khronos.org/c/gltf-general)
 
-glTF Experience Format (glXF) is the specification for a new file type in the glTF ecosystem for efficient composition of multiple glTF assets.
+glTF Reference Format (glTFX - formely called "glTF Experience Format (glXF)") is the specification for a new file type in the glTF ecosystem for efficient composition of multiple glTF assets.
```

Comment:
Will the `"experience": true` property inside of `"asset"` be changed to `"reference": true`?